src/sender/sender.service.ts

```diff
@@ -0,0 +1,24 @@
+import { IUtilsServices } from "..";
+import { IDateHandlerService } from "../date-handler/date-handler.service";
+import { IInvoice } from "../interfaces/invoice.interface";
+
+export interface ISenderService {
+  informAboutNewInvoice: (invoice: IInvoice) => Promise<string>;
+}
+
+class SenderService {
+  private readonly dateHandlerService: IDateHandlerService;
+  constructor({ dateHandlerService }: IUtilsServices) {
+    this.dateHandlerService = dateHandlerService;
+  }
+
+  async informAboutNewInvoice(invoice: IInvoice) {
+    return `New invoice from: ${
+      invoice.owner
+    } has been issued.\nAmount to pay: ${
+      invoice.amountDue
+    }, due date: ${this.dateHandlerService.getFullDate(invoice.dueDate)}`;
+  }
+}
```

Comment:
Czy to koniecznie powinno zwracać stringa?

W przyszłości prawdopodobnie będzie to void..

Reply:
jest void